nautobot/core/tests/runner.py

```diff
@@ -52,3 +51,133 @@ def teardown_databases(self, old_config, **kwargs):
             print("Database emptied!")
 
         super().teardown_databases(old_config, **kwargs)
+
+
+# Use django_slowtests only when GENERATE_PERFORMANCE_REPORT flag is set to true
+if settings.TEST_GENERATE_PERFORMANCE_REPORT:
+    import yaml
+
+    try:
+        from django_slowtests.testrunner import DiscoverSlowestTestsRunner
+
+        print("Using NautobotPerformanceTestRunner to run tests ...")
+    except ImportError:
+        print("Unable to use NautobotPerformanceTestRunner to run tests. Is the 'django_slowtests' package installed?")
+
+    class NautobotPerformanceTestRunner(NautobotTestRunner, DiscoverSlowestTestsRunner):
```

Comment:
Is it necessary to define this entire class conditionally? Don't we still need to overload the `TEST_RUNNER` setting (or pass `--testrunner` to `nautobot-server test`) to activate it? Unless I'm misunderstanding?

Reply:
I confuse myself sometimes when Im implementing something but you are absolutely right. The try and except block should cover it.